tests/test_asciiblock.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018 Richard Hull and contributors
+# See LICENSE.rst for details.
+
+"""
+Tests for :py:class:`luma.emulator.device.asciiblock`.
+"""
+
+import hashlib
+import struct
+import sys
+try:
```

Comment:
@rm-hull we do not support older versions of Python so the `io` version should be enough.

Reply:
Get some strange unicode encoding problems on py27 using io.StringIO:

```
self = <luma.emulator.device.asciiblock object at 0x10f634350>, cmd = '2J'

    def _CSI(self, cmd):
        """
            Control sequence introducer
            """
>       sys.stdout.write('\x1b[')
E       TypeError: unicode argument expected, got 'str'
```

Reply:
So i can use io.BytesIO instead for py27